data/actionsmap/yunohost.yml

```diff
@@ -1351,6 +1351,16 @@ tools:
                     help: Show private data (domain, IP)
                     action: store_true
 
+        ### tools_portavailable()
+        portavailable:
```

Comment:
Following discussions about this (https://dev.yunohost.org/issues/872), let's rename this 'port-available' :wink: